oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -364,6 +364,42 @@ void testGetCommandLine() {
                 is(greaterThan(0)));
     }
 
+    @Test
+    void testGetArguments() {
+        int processesWithNonEmptyArguments = 0;
+
+        SystemInfo si = new SystemInfo();
+        OperatingSystem os = si.getOperatingSystem();
+        for (OSProcess process : os.getProcesses(null, null, 0)) {
```

Comment:
Why not create a `Predicate` for testing empty args and pass it to `getProcesses()`.  Then you just return the list size.  Yeah, overkill for a test method, I guess.  But still, behold the elegance!
```
int procsWithArgs = os.getProcesses(p -> !p.getArguments().isEmpty(), null, 0).size();
```
Better yet instead of asserting the size is greater than 0, roll up the above into:
```
assertThat("Processes with non-empty arguments should be 1 or higher",
    os.getProcesses(p -> !p.getArguments().isEmpty(), null, 0), is(not(empty())));
```